inst/rmarkdown/templates/template_sfp_en/skeleton/01_dependencies.Rmd

```diff
@@ -1,7 +1,24 @@
 # Dependencies
 
 <!--
-Alleen in te vullen indien van toepassing.
-Protocols waarnaar in dit protocol verwezen wordt: <protocol-code>-YYYY.NN.
-Protocols die verwijzen naar dit protocol: <protocol-code>-YYYY.NN.
+Hier niets invullen.
+Indien er dependencies zijn deze invullen in de yaml sectie van index.Rmd
 -->
+
+
+```{r dependencies}
+vn <- unlist(params$dependencies_versionnumber)
+pm <- unlist(params$dependencies_params)
+ap <- as.logical(unlist(params['dependencies_appendix']))
+
+dependencies <- as_tibble(params$dependencies_protocolcode) %>%
+  rename(protocol_code = value) %>%
+  mutate(version_number = vn,
+         params = pm,
+         appendix = ap
+         )
+
+dependencies %>%
+  select(-appendix) %>%
```

Comment:
Still would be nice to have a column 'included as appendix' I think.

Reply:
done